### PR TITLE
docs: translate to Spanish publish-python-package-pypi-conda.md

### PR DESCRIPTION
**Pull request by RobPasMue**
Related to #313

### CODE REVIEW

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 1]
```diff
@@ -1705,39 +1705,49 @@ msgstr ""
...
 "Python package."
 msgstr ""
+"A continuación, aprenderá más sobre las diversas opciones de publicación para su paquete de Python."
 
 #: ../../package-structure-code/publish-python-package-pypi-conda.md:14
 msgid ""
 "Installing packages in the same environment using both pip and conda can "
 "lead to package conflicts."
 msgstr ""
+"Instalar paquetes en el mismo entorno usando tanto pip como conda puede "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1732]: Looks like we are missing a line here

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 2]
```diff
@@ -1749,6 +1759,11 @@ msgid ""
 "distributions. From PyPI if you create a conda-forge recipe you can then "
 "publish to conda-forge."
 msgstr ""
+"Imagen representando la progresión de crear un paquete de Python, construirlo"
+" y luego publicarlo en PyPI y conda-forge. Toma tu código y conviertelo en "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1763]:
```suggestion
" y luego publicarlo en PyPI y conda-forge. Convierte tu código en "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 3]
```diff
@@ -1758,10 +1773,14 @@ msgid ""
 "your package on conda-forge. You do not need to rebuild your package to "
 "publish to conda-forge."
 msgstr ""
+"Una vez que haya publicado ambas distribuciones de paquetes (la distribución de origen "
+"y la rueda) en PyPI, puede publicar en conda-forge. conda-forge requiere una distribución de "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1777]:
```suggestion
"y la wheel) en PyPI, puede publicar en conda-forge. conda-forge requiere una distribución de "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 4]
```diff
@@ -1770,23 +1789,31 @@ msgid ""
...
+"se pueden instalar usando `pip`."
 
 #: ../../package-structure-code/publish-python-package-pypi-conda.md:38
 msgid ""
 "The biggest different between using pip and conda to install a package is"
 " that conda can install any package regardless of the language(s) that it"
 " is written in. Whereas `pip` can only install Python packages."
 msgstr ""
+"La mayor diferencia entre usar pip y conda para instalar un paquete es que conda puede instalar "
+"cualquier paquete independientemente del idioma(s) en el que esté escrito. Mientras que `pip` "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1811]:
```suggestion
"cualquier paquete independientemente del lenguaje en el que esté escrito. Mientras que `pip` "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 5]
```diff
@@ -1852,23 +1898,31 @@ msgid ""
 "that mixes all of these packages is usually easier and more consistent "
 "with full-fledged package managers like conda."
 msgstr ""
+"Si bien conda se creó originalmente para admitir paquetes de Python, ahora se usa en todos los lenguajes. "
+"Este soporte entre lenguajes facilita que algunos paquetes incluyan y tengan acceso a herramientas escritas "
+"en otros lenguajes, como C/C++ (gdal), Julia o R. Crear un entorno que mezcle todos estos paquetes suele ser "
+"más fácil y más consistente con administradores de paquetes completos como conda."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1904]:
```suggestion
"más fácil y más consistente con herramientas de administración de paquetes como conda."
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 6]
```diff
@@ -1909,6 +1972,13 @@ msgid ""
 "says PyPI servers. PyPI - anyone can publish to PyPI. and test PyPI. a "
 "testbed server for you to practice."
 msgstr ""
+"Gráfico con el título Repositorios de paquetes de Python. Debajo dice "
+"Cualquier cosa alojada en PyPI se puede instalar usando pip install. Los paquetes alojados en un "
+"canal de conda se pueden instalar usando conda install. Debajo de eso hay dos filas. La fila superior "
+"dice canales de conda. al lado hay tres cuadros uno con conda-forge, mantenido por la comunidad; bioconda "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:1978]:
```suggestion
"dice canales de conda. Al lado hay tres cuadros uno con conda-forge, mantenido por la comunidad; bioconda "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 7]
```diff
@@ -1969,12 +2055,19 @@ msgid ""
...
+"de conflictos de dependencias cuando las herramientas se instalan desde el mismo repositorio de paquetes. "
+"Por lo tanto, los entornos que contienen paquetes instalados tanto desde pip como desde conda tienen más "
+"probabilidades de generar conflictos de dependencias."
 
 #: ../../package-structure-code/publish-python-package-pypi-conda.md:144
 msgid ""
 "Similarly installing packages from the default anaconda package mixed "
 "with the conda-forge channel can also lead to dependency conflicts."
 msgstr ""
+"De manera similar, instalar paquetes del paquete predeterminado de Anaconda mezclado con el canal conda-forge "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:2069]:
```suggestion
"De manera similar, instalar paquetes del canal predeterminado (`defaults`) de Anaconda mezclado con el canal conda-forge "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 8]
```diff
@@ -2000,32 +2101,41 @@ msgid ""
...
-msgstr ""
+msgstr "Como publicar en conda-forge"
 
 #: ../../package-structure-code/publish-python-package-pypi-conda.md:167
 msgid ""
 "While pyOpenSci doesn't require you to add your package to conda-forge, "
 "we encourage you to consider doing so!"
 msgstr ""
+"¡Si bien pyOpenSci no requiere que agregue su paquete a conda-forge, le "
+"animamos a que lo considere!"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:2138]:
```suggestion
"sugerimos que lo considere!"
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 9]
```diff
@@ -2077,6 +2200,11 @@ msgid ""
 "file found in the pull request match the PyPI metadata of the new "
 "release."
 msgstr ""
+"Puede fusionar la solicitud de cambio para esa actualización una vez que esté "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:2203]:
```suggestion
"Puede hacer merge de la pull request para esa actualización una vez que esté "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 10]
```diff
@@ -2077,6 +2200,11 @@ msgid ""
...
+"contento con ella. Una solicitud de cambio lista para fusionar generalmente "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:2204]:
```suggestion
"contento con ella. Una pull request lista para hacer merge generalmente "
```

**Review by flpm — COMMENTED** [review]
Just some minor comments.

File: locales/es/LC_MESSAGES/package-structure-code.po (hunk 1 above)
Comment (RobPasMue) [reply locales/es/LC_MESSAGES/package-structure-code.po:1732]: Yep- my bad! on it
```suggestion
"Instalar paquetes en el mismo entorno usando tanto pip como conda puede "
"dar lugar a conflictos entre paquetes."
```

**Comment by RobPasMue** [comment]
Thanks again @flpm for your review!

**Review by flpm — APPROVED** [review]
LGTM

**Pull request merged**